Classes/Validation/Validator/RegistrationFieldValidator.php

```diff
@@ -33,7 +33,7 @@ class RegistrationFieldValidator extends AbstractValidator
     protected function isValid($registration)
     {
         $result = true;
-        if ($registration->getEvent() === null || ($registration->getFieldValues()->count() === 0 &&
+        if ($registration->getEvent() === null || ($registration->getFieldValues()?->count() === 0 &&
```

Comment:
The nullsafe operator is a PHP 8 only feature, so this can not be used in version 6.x of the extension, since it is compatible with PHP 7 and 8.